packages/nimble-components/src/wafer-map/modules/zoom-handler.ts

```diff
@@ -20,9 +20,11 @@ export class ZoomHandler {
     private readonly minScale = 1.1;
     private readonly minExtentPoint: [number, number] = [-100, -100];
     private readonly extentPadding = 100;
-    private readonly zoomBehavior: ZoomBehavior<Element, unknown>;
+    private zoomBehavior!: ZoomBehavior<Element, unknown>;
 
-    public constructor(private readonly wafermap: WaferMap) {
+    public constructor(private readonly wafermap: WaferMap) {}
+
+    public updateZoomBehavior(): void {
         this.zoomBehavior = this.createZoomBehavior();
         this.zoomBehavior(select(this.wafermap.canvas as Element));
     }
```

Comment:
Similar to the event cordinator can you just make a public createZoomBehavior after merging together updateZoomBehavior and createZoomBehavior.

I recognize other changes are more of an update pattern but this case is more of a create than an update